code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -496,7 +496,7 @@
 /obj/item/clothing/suit/space/changeling/process()
 	if(ishuman(loc))
 		var/mob/living/carbon/human/H = loc
-		H.reagents.add_reagent(/datum/reagent/medicine/salbutamol, REAGENTS_METABOLISM)
+		H.reagents.add_reagent(/datum/reagent/medicine/salbutamol, REAGENTS_METABOLISM * (SSOBJ_DT / SSMOBS_DT))
```

Comment:
why would this one need 2 DTs? it affects mobs but it runs on the objects process() so you only need SSOBJ_DT

Reply:
Since it uses reagents_metabolism constant I assume it's supposed to fill just enough for the mob to metabolize. Thus both DTs are needed.

Reply:
right, makes sense.